attest/ake/src/responder.rs

```diff
@@ -82,13 +82,19 @@ impl ResponderTransitionMixin for Start {
     fn handle_response<KexAlgo, Cipher, DigestAlgo>(
         csprng: &mut (impl CryptoRng + RngCore),
         handshake_state: HandshakeState<KexAlgo, Cipher, DigestAlgo>,
-        ias_report: VerificationReport,
+        attestation_evidence: EvidenceKind,
     ) -> Result<(Ready<Cipher>, AuthResponseOutput), Error>
     where
         KexAlgo: Kex,
         Cipher: NoiseCipher,
         DigestAlgo: NoiseDigest,
     {
+        // TODO: This will be replaced with dcap evidence serialization.
+        //       This code will never run.
+        let ias_report = match attestation_evidence {
+            EvidenceKind::Epid(verification_report) => verification_report,
+            _ => Err(Error::AttestationEvidenceSerialization)?,
+        };
```

Comment:
More of an FYI on [let-else](https://doc.rust-lang.org/rust-by-example/flow_control/let_else.html)
This will get changed on the next PR that brings in DCAP evidence so it can be converted then if people prefer let-else
```suggestion
        let EvidenceKind::Epid(ias_report) = attestation_evidence else {
            return Err(Error::AttestationEvidenceSerialization);
        };
```